bindings/cs/MongoDB.Libmongocrypt.Test/Callbacks/CallbackUtils.cs

```diff
@@ -0,0 +1,32 @@
+﻿/*
+ * Copyright 2020–present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Linq;
+
+namespace MongoDB.Libmongocrypt.Test.Callbacks
+{
+    internal static class CallbackUtils
+    {
+        public static byte[] GetBytesFromHex(string hex)
+        {
+            return Enumerable.Range(0, hex.Length)
```

Comment:
There are more efficient solutions on this page https://stackoverflow.com/questions/311165/how-do-you-convert-a-byte-array-to-a-hexadecimal-string-and-vice-versa

then https://stackoverflow.com/questions/321370/how-can-i-convert-a-hex-string-to-a-byte-array